src/ClientHandle.cpp

```diff
@@ -839,8 +839,15 @@ void cClientHandle::HandlePlayerPos(double a_PosX, double a_PosY, double a_PosZ,
 
 	Vector3d NewPosition(a_PosX, a_PosY, a_PosZ);
 	Vector3d OldPosition = GetPlayer()->GetPosition();
+	double OldStance = GetPlayer()->GetStance();
 	auto PreviousIsOnGround = GetPlayer()->IsOnGround();
 
+	if ((OldPosition == NewPosition) && (OldStance == a_Stance))
```

Comment:
Why are you not checking `a_IsOnGround` as well?
```suggestion
	if (
	    (OldPosition == NewPosition) &&
	    (OldStance == a_Stance) &&
	    (PreviousIsOnGround == a_IsOnGround)
	)
```